ods_ci/build/ldap/ldap_installation.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+LDAP_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+oc create secret generic ldap-bind-password --from-literal=bindPassword=adminpassword -n openshift-config || echo "ldap secret exists"
+oc apply -f $LDAP_PATH/ldap.yaml
+sleep 25s
```

Comment:
Maybe we should sample the `openldap` resource until it exists?

Reply:
It is good to have it, for now we just port it from the existing code.